index.src.html

```diff
@@ -548,6 +548,34 @@ <h1>Credential Management Level 1</h1>
     to guide the discovery process (by choosing a federated identity provider, BTLE device, etc).
   </div>
 
+  <div algorithm="copy any buffer sources">
+    To <dfn>create a copy of any buffer sources' data</dfn> referenced by a value |V| of
+    [=dictionary=] type, run the following steps given |V|:
+
+    <ol class="algorithm">
+		1. Let |dictionaries| be a list consisting of |V| and all of |V|'s 
+			[=inherited dictionaries=], in order from least to most derived.
+
+		2. For each dictionary |dictionary| in |dictionaries|, in order:
+
+			1. For each dictionary member |member| declared on 
+				 |dictionary|, in lexicographical order:
```

Comment:
Rather than walking through the dictionaries, did you consider just walking through |V|'s members?